src/test/java/org/reaktivity/nukleus/maven/plugin/internal/generated/ListWithPhysicalAndLogicalLengthFW.java

```diff
@@ -0,0 +1,330 @@
+    {
+        return offset() + buffer().getInt(offset() + PHYSICAL_LENGTH_OFFSET) & 0xFFFF;
+    }
+
+    @Override
+    public String toString()
+    {
+        return String.format("LIST_WITH_PHYSICAL_AND_LOGICAL_LENGTH [field0=%s, field1=%d, field2=%s]",
+            field0() != null ? field0RO.asString() : null, field1(), field2() != null ? field2RO.asString() : null);
+    }
+
+    public static final class Builder extends Flyweight.Builder<ListWithPhysicalAndLogicalLengthFW>
+    {
+        private final StringFW.Builder field0RW = new StringFW.Builder();
+
+        private final StringFW.Builder field2RW = new StringFW.Builder();
+
+        private int fieldsMask;
+
+        public Builder()
+        {
+            super(new ListWithPhysicalAndLogicalLengthFW());
+        }
+
+        private StringFW.Builder field0()
+        {
+            return field0RW.wrap(buffer(), limit(), maxLimit());
+        }
+
+        public Builder field0(
+            String value)
+        {
+            assert (fieldsMask & ~0x00) == 0 : "Field \"field0\" is already set or subsequent fields are already set";
+            StringFW.Builder field0RW = field0();
+            field0RW.set(value, StandardCharsets.UTF_8);
+            fieldsMask |= 1 << FIELD_INDEX_FIELD0;
+            limit(field0RW.build().limit());
+            return this;
+        }
+
+        public Builder field0(
+            StringFW value)
+        {
+            assert (fieldsMask & ~0x00) == 0 : "Field \"field0\" is already set or subsequent fields are already set";
+            StringFW.Builder field0RW = field0();
+            field0RW.set(value);
+            fieldsMask |= 1 << FIELD_INDEX_FIELD0;
+            limit(field0RW.build().limit());
+            return this;
+        }
+
+        public Builder field0(
+            DirectBuffer buffer,
+            int offset,
+            int length)
+        {
+            assert (fieldsMask & ~0x00) == 0 : "Field \"field0\" is already set or subsequent fields are already set";
+            StringFW.Builder field0RW = field0();
+            field0RW.set(buffer, offset, length);
+            fieldsMask |= 1 << FIELD_INDEX_FIELD0;
+            limit(field0RW.build().limit());
+            return this;
+        }
+
+        public Builder field1(
+            long value)
+        {
+            assert (fieldsMask & ~0x01) == 0 : "Field \"field1\" is already set or subsequent fields are already set";
+            assert (fieldsMask & 0x01) != 0 : "Prior required field \"field0\" is not set";
+            if (value < 0)
+            {
+                throw new IllegalArgumentException(String.format("Value %d too low for field \"field1\"", value));
+            }
+            assert (value & 0xffff_ffff_0000_0000L) == 0L : "Value out of range for field \"field1\"";
+            int newBit = fieldsMask | (1 << FIELD_INDEX_FIELD1);
+            assert newBit == (newBit & 0x03) : "Value out of order for field \"field1\" in the list";
```

Comment:
Do we still need this out of order check, or can we remove it after adding those asserts above?